services/efs-guardian/src/simcore_service_efs_guardian/core/application.py

```diff
@@ -36,29 +39,30 @@ def create_app(settings: ApplicationSettings) -> FastAPI:
         redoc_url=None,  # default disabled
     )
     # STATE
-    app.state.settings = settings
+    app.state.settings = app_settings
     assert app.state.settings.API_VERSION == API_VERSION  # nosec
     if app.state.settings.EFS_GUARDIAN_TRACING:
         setup_tracing(app, app.state.settings.EFS_GUARDIAN_TRACING, APP_NAME)
 
     # PLUGINS SETUP
     setup_rabbitmq(app)
     setup_redis(app)
+    setup_db(app)
 
     setup_api_routes(app)
-    setup_rpc_routes(app)
+    setup_rpc_routes(app)  # requires Rabbit
```

Comment:
THOUGHT: it would be good to setup a mechanism that enforces this dependencies. Similar to the one in web-server.